pkg/subctl/cmd/validate_cni.go

```diff
@@ -73,5 +86,77 @@ func validateCniConfig(cmd *cobra.Command, args []string) {
 			submariner.Status.NetworkPlugin)
 		status.QueueSuccessMessage(message)
 		status.End(cli.Success)
+
+		if submariner.Status.NetworkPlugin == "canal-flannel" {
+			validateIPPool(item.config, submariner)
+		}
+	}
+}
+
+func validateIPPool(config *rest.Config, submariner *v1alpha1.Submariner) {
+	dynClient, err := dynamic.NewForConfig(config)
+	exitOnError("Error creating clients for cluster", err)
+
+	client := dynClient.Resource(calicoGVR)
+
+	if err != nil {
+		exitOnError("Error creating Kubernetes client", err)
+	}
+
+	message := "Validating that Calico IPPools configuration"
+	status.Start(message)
+
+	findClusterCIDR := false
+	findServiceCIDR := false
+	ippools, err := client.List(metav1.ListOptions{})
+	if err != nil {
+		message := fmt.Sprintf("Error obtaining IPPools: %v", err)
+		status.QueueFailureMessage(message)
+		status.End(cli.Failure)
+	}
+
+	if len(ippools.Items) < 1 {
+		message := "IPPool is missing"
+		status.QueueFailureMessage(message)
+		status.End(cli.Failure)
```

Comment:
Is it a failure if there are no IPPools?

```suggestion
		status.QueueFailureMessage("No IPPools found")
		status.End(cli.Failure)
		return
```